frontend/package.json

```diff
@@ -9,23 +9,17 @@
     "postinstall": "symlink-dir node_modules/cesium/Build/Cesium public/cesium"
   },
   "dependencies": {
-    "bootstrap": "5.3.3",
-    "cesium": "1.124.0",
-    "next": "15.2.3",
+    "bootstrap": "^5.3.3",
+    "cesium": "^1.124.0",
+    "next": "^15.2.3",
     "prop-types": "^15.8.1",
-    "react": "18.3.1",
-    "react-bootstrap": "2.10.6",
-    "react-dom": "18.3.1",
+    "protobufjs": "^7.4.0",
+    "react": "^18.3.1",
+    "react-bootstrap": "^2.10.6",
+    "react-dom": "^18.3.1",
     "resium": "^1.18.2"
```

Comment:
So resium (the react wrapper arround cesium), latest full release is 1.18.3, which support cesium 1.116.0 (having 1.124.0 current may be an issue) and react 18.x. Going past those version gets into unstable territory (cesium's api usually changes on each release and react 19 is not supported currently). Since resium version can effect both cesium and react versions, I would not include the carrot for cesium and react version.

There is a 1.19.0-beta release for resium that we could use that suppot react 19 and cesium 1.126.0 we could try.